cypress/integration/upload.spec.ts

```diff
@@ -10,42 +10,24 @@ describe("Upload from the the landing page and Download KMP", function () {
   });
 
   beforeEach(() => {
+    cy.clearLocalData();
     cy.task("clearDownloads");
     cy.allowUnlimitedDownloadsToFolder(downloadFolder);
-    cy.visit("/");
   });
 
   it("should find a button to press to upload a file", function () {
+    // Load languages
+    cy.visit("/");
+    cy.wait(500);
```

Comment:
Are these waits for `cy.intercept()`?

Reply:
This wait is for the race conditions I'm experiencing. While I cannot get a direct error message, it seems that there are moments where the app will not load things in time from the IndexedDB. This behaviour didn't occur before, so I'll see if I need to make the clearing and generation of the IndexedDB blocking or something of the sort

Reply:
Yes, these waits are for `cy.intercept()`. I'm fairly sure the reason why these `wait()`s are required now is that the keyman keyboards get cleared on every test. Since the keyboard gets cleared, an API request needs to be made and get intercepted, which can take milliseconds to complete.

What I will try later today is to make a command that clears everything _except_ the keyman keyboards. Hopefully this works as a fair compromise between the two solutions we have